File: src/main/java/casino/dwight/coc/entities/BadgeUrls.java
package casino.dwight.coc.entities;

import com.fasterxml.jackson.annotation.JsonProperty;

public class BadgeUrls {
    private String small;
    private String large;
    private String medium;

    @JsonProperty("small")
    public String getSmall() { return small; }
    @JsonProperty("small")
    public void setSmall(String value) { this.small = value; }

    @JsonProperty("large")
    public String getLarge() { return large; }
    @JsonProperty("large")
    public void setLarge(String value) { this.large = value; }

    @JsonProperty("medium")
    public String getMedium() { return medium; }
    @JsonProperty("medium")
    public void setMedium(String value) { this.medium = value; }
}

File: Clash/Entities/BadgeUrls.cs
using System;
using Model = ClashWrapper.Models.BadgeUrlModel;

namespace ClashWrapper.Entities
{
    public class BadgeUrls
    {
        private readonly Model _model;

        internal BadgeUrls(Model model)
        {
            _model = model;
        }

        private Uri _small;
        public Uri Small => _small ?? (_small = new Uri(_model.Small));

        private Uri _medium;
        public Uri Medium => _medium ?? (_medium = new Uri(_model.Medium));

        private Uri _large;
        public Uri Large => _large ?? (_large = new Uri(_model.Large));
    }
}
